geometry/create-and-edit-geometries/src/main/java/com/esri/samples/create_and_edit_geometries/CreateAndEditGeometriesController.java

```diff
@@ -0,0 +1,591 @@
+ * the License.
+ */
+
+package com.esri.samples.create_and_edit_geometries;
+
+import com.esri.arcgisruntime.ArcGISRuntimeEnvironment;
+import com.esri.arcgisruntime.concurrent.ListenableFuture;
+import com.esri.arcgisruntime.geometry.GeometryType;
+import com.esri.arcgisruntime.geometry.MultipointBuilder;
+import com.esri.arcgisruntime.geometry.Point;
+import com.esri.arcgisruntime.geometry.PointBuilder;
+import com.esri.arcgisruntime.geometry.PointCollection;
+import com.esri.arcgisruntime.geometry.PolylineBuilder;
+import com.esri.arcgisruntime.geometry.PolygonBuilder;
+import com.esri.arcgisruntime.geometry.SpatialReferences;
+import com.esri.arcgisruntime.mapping.ArcGISMap;
+import com.esri.arcgisruntime.mapping.BasemapStyle;
+import com.esri.arcgisruntime.mapping.Viewpoint;
+import com.esri.arcgisruntime.mapping.view.Graphic;
+import com.esri.arcgisruntime.mapping.view.GraphicsOverlay;
+import com.esri.arcgisruntime.mapping.view.IdentifyGraphicsOverlayResult;
+import com.esri.arcgisruntime.mapping.view.MapView;
+import com.esri.arcgisruntime.mapping.view.geometryeditor.FreehandTool;
+import com.esri.arcgisruntime.mapping.view.geometryeditor.GeometryEditor;
+import com.esri.arcgisruntime.mapping.view.geometryeditor.GeometryEditorTool;
+import com.esri.arcgisruntime.mapping.view.geometryeditor.VertexTool;
+import com.esri.arcgisruntime.mapping.view.geometryeditor.GeometryEditorElement;
+import com.esri.arcgisruntime.symbology.SimpleFillSymbol;
+import com.esri.arcgisruntime.symbology.SimpleLineSymbol;
+import com.esri.arcgisruntime.symbology.SimpleMarkerSymbol;
+
+import javafx.beans.property.SimpleBooleanProperty;
+import javafx.beans.value.ChangeListener;
+import javafx.fxml.FXML;
+import javafx.geometry.Point2D;
+import javafx.scene.control.Button;
+import javafx.scene.control.ComboBox;
+import javafx.scene.paint.Color;
+import javafx.util.StringConverter;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class CreateAndEditGeometriesController {
+
+
+  @FXML
+  private MapView mapView;
+  @FXML
+  private Button pointButton;
+  @FXML
+  private Button multipointButton;
+  @FXML
+  private Button polylineButton;
+  @FXML
+  private Button polygonButton;
+  @FXML
+  private Button redoButton;
+  @FXML
+  private Button undoButton;
+  @FXML
+  public Button deleteSelectedElementButton;
+  @FXML
+  private Button stopAndSaveButton;
+  @FXML
+  private Button stopAndDiscardButton;
+  @FXML
+  public Button deleteAllGeometriesButton;
+  @FXML
+  private ComboBox<GeometryEditorTool> toolComboBox;
+
+  private GeometryEditor geometryEditor;
+  private VertexTool vertexTool;
+  private FreehandTool freehandTool;
+
+  private GraphicsOverlay graphicsOverlay;
+  private Graphic selectedGraphic;
+
+  private SimpleFillSymbol fillSymbol;
+  private SimpleLineSymbol lineSymbol;
+  private SimpleMarkerSymbol pointSymbol;
+  private SimpleMarkerSymbol multipointSymbol;
```

Comment:
```suggestion
  private GeometryEditor geometryEditor;
  private VertexTool vertexTool;
  private FreehandTool freehandTool;
  private GraphicsOverlay graphicsOverlay;
  private Graphic selectedGraphic;
  private SimpleFillSymbol fillSymbol;
  private SimpleLineSymbol lineSymbol;
  private SimpleMarkerSymbol pointSymbol;
  private SimpleMarkerSymbol multipointSymbol;
```
These don't need to be separated out, though I understand why you did.